doc/man/salt.7

```diff
@@ -80063,6 +80063,10 @@ my\-ec2\-config:
   # Optional
   ssh_gateway_username: root
 
+  # Default to nc -q0 %h %p
+  # Optional
+  ssh_gateway_command: "-W %h:%p"
+
```

Comment:
Can you remove this change, we compile these every time we build, so this will not remain.

Reply:
thanks a lot. I will update `ec2.py` and `misc.rst` instead :)